pandas/tests/indexes/test_interval.py

```diff
@@ -491,6 +649,207 @@ def testcontains(self):
         assert not i.contains(20)
         assert not i.contains(-20)
 
+        # Why are there two tests called test contains? this should be cleared up a little, no?
+
+
+    @pytest.mark.xfail(reason="new indexing tests for issue 16316")
+    def test_interval_covers_interval_updated_behavior(self):
+
+        # class Interval:
+        #     def covers(self, other: Interval) -> bool
+
+        assert     Interval(1, 3).covers(Interval(1.5, 2.5))
+        assert     Interval(1, 3).covers(Interval(1, 2))
+        assert     Interval(1, 3).covers(Interval(2, 3))
+        assert not Interval(1, 3).covers(Interval(0.5, 2.5))
+        assert not Interval(1, 3).covers(Interval(1.5, 3.5))
+
+        assert     Interval(1, 3, closed='right').covers(Interval(1, 3, closed='right'))
+        assert not Interval(1, 3, closed='right').covers(Interval(1, 3, closed='left'))
```

Comment:
those lines are also too long for PEP8 (if you look at the travis log of the third output, all at the bottom, you can see all PEP8 violations, but I would recommend configuring your editor to also warn for this). 

See http://pandas-docs.github.io/pandas-docs-travis/contributing.html#python-pep8